scrapers/DDFNetwork.yml

```diff
@@ -0,0 +1,27 @@
+name: DDFNetwork
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - ddfnetwork.com/videos/
+      - ddfbusty.com/videos/
+      - 1by-day.com/videos/
+      - eurogirlsongirls.com/videos/
+      - euroteenerotica.com/videos/
+      - handsonhardcore.com/videos/
+      - hotlegsandfeet.com/videos/
+      - houseoftaboo.com/videos/
+      - onlyblowjob.com/videos/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //meta[@itemprop="name"]/@content
+      Date: //meta[@itemprop="uploadDate"]/@content
+      Details: //meta[@name="description"]/@content
```

Comment:
the details selector seems to get partial description for all sites except pornworld.com
this seems to work ( got the desc for pornworld from a div just to be safe)
```yaml
      Details: //div/dt[contains(.,"Description:")]/../dd|//div[@class="descr-box"]/p
```

Reply:
Tested on my side, it look like the xPath from DDFNetwork.yml don't work for Pornworld.
Are you sure that you don't used the pornworld.yml when you tested it ?

Reply:
oh you are absolutely right, i tottalyy forgot about the other scraper